kbhff/api/email.py

```diff
@@ -2,38 +2,56 @@
 from kbhff.api.exceptions import *
 import time
 
-# Try to get mail credentials
-try:
-    # check if module exists:
-    from kbhff.api.mail_credentials import *
-except ModuleNotFoundError:
-    # check if environment variables exist:
-    from os import environ
-    mail_credentials = {}
-    mail_credentials["login"] = environ.get("MAIL_CREDENTIALS_EMAIL")
-    mail_credentials["password"] = environ.get("MAIL_CREDENTIALS_PASSWORD")
-
+def get_mail_credentials():
+    try:
+        # check if module exists:
+        from kbhff.api.mail_credentials import mail_credentials
+    except ModuleNotFoundError:
+        # check if environment variables exist:
+        from os import environ
+        mail_credentials = {}
+        # values will be None if environment variable doesn't exist
+        mail_credentials["login"] = environ.get("MAIL_CREDENTIALS_EMAIL")
+        mail_credentials["password"] = environ.get("MAIL_CREDENTIALS_PASSWORD")
```

Comment:
If this returns (None, None), will the program raise an error at a later point? If so, it would make more sense to check for that and fail early.

Reply:
I thought about prompting the user to input credentials at some point but think it's a bit impractical and unnecessary for now.
I'll make them throw an eror.